flow360/component/simulation/validation/validation_simulation_params.py

```diff
@@ -47,6 +52,57 @@ def _check_consistency_wall_function_and_surface_output(v):
     return v
 
 
+def _check_duplicate_entity_in_model(params):
+    models = params.models
+
+    dict_entity = {"Surface": {}, "Volume": {}}
+    registry_entity = {"Surface": EntityRegistry(), "Volume": EntityRegistry()}
+
+    is_duplicate = False
+
+    if models:
+        for model in models:
+            if hasattr(model, "entities"):
+                for entity in model.entities.stored_entities:
+                    registry_entity, dict_entity, is_duplicate = _register_single_entity(
+                        entity, model.type, registry_entity, dict_entity, is_duplicate
+                    )
+
+    if is_duplicate:
+        error_msg = ""
+        for entity_type in dict_entity.keys():
+            for entity_name, (model_list, is_invalid_entity) in dict_entity[entity_type].items():
+                if is_invalid_entity:
+                    model_string = ",".join(f"`{x}`" for x in model_list)
```

Comment:
`model_string` : What if there are 2 same type model? E.g. Rotation and Rotation?

Reply:
Currently, it will show "Volume Entity {name} appears multiple time in Rotation model." Is this okay?